src/sql_engine/src/query/bind.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright 2020 Alex Dukhno
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use bigdecimal::BigDecimal;
+use protocol::{results::QueryError, sql_values::PostgreSqlValue, Sender};
+use sqlparser::ast::{Assignment, Expr, Ident, Query, SetExpr, Statement, Value};
+use std::sync::Arc;
+
+type Result = std::result::Result<(), ()>;
+
+pub(crate) struct ParamBinder {
+    sender: Arc<dyn Sender>,
+}
+
+impl ParamBinder {
```

Comment:
I think we need one more enhancement task to make this struct public and use in `node` module directly, at least this approach should be investigated.

No need to change anything for this comment.